web/app/js/components/Navigation.jsx

```diff
@@ -166,12 +174,42 @@ class NavigationBase extends React.Component {
       }).catch(this.handleApiError);
   }
 
+  fetchLatestCommunityUpdate() {
+    let jsonFeedUrl = "https://linkerd.io/dashboard/index.json";
+    this.communityUpdatesPromise = fetch(jsonFeedUrl)
+      .then(rsp => rsp.json())
+      .then(rsp => rsp.data.date)
+      .then(rsp => {
+        if (rsp.length > 0) {
+          let lastClicked = localStorage["linkerd-updates-last-checked"];
+          if (!lastClicked) {
+            localStorage.setItem("linkerd-updates-last-checked", new Date());
```

Comment:
just want to confirm that we're intending to set `linkerd-updates-last-checked` on the date of the last page load, not the date of the last time the `Community` nav item was clicked. if so, considering renaming `lastClicked` to `lastChecked`.

related to this, users will typically view this page via `linkerd dashboard`, so the port will be different each time, causing the local storage to be empty and the badge to always show up. we've talked about pinning the dashboard port rather than random port assignment, but that change is probably outside the scope of this PR. i'm fine with this as-is just wanted to call it out.

Reply:
re: `linkerd-updates-last-checked`, thanks for the questions, i realized i was overcomplicating the logic. i've now simplified so that the only place where `localStorage` is set is in `handleCommunityClick()`. 

re: port assignments, thanks for pointing this out. 🔍🕵️  as we talked about in person, will make a separate PR for this.

Reply:
Addressed in PR #2486